diozero-core/src/main/java/com/diozero/internal/provider/builtin/NativeGpioInputOutputDevice.java

```diff
@@ -117,16 +117,18 @@ public int getGpio() {
 	public boolean getValue() throws RuntimeIOException {
 		if (mmapGpio == null) {
 			return line.getValue() == 0 ? false : true;
+		} else {
```

Comment:
This else is not necessary, as the `return` in the `if` clause will exit the function.

Reply:
cosmetic change to make the code of getValue() and setValue() look similar